index.html

```diff
@@ -203,7 +215,7 @@ <h2>Profesional Art <br />Printing Data</h2>
                         <a href="#">RUBY</a>
                     </li>
                 </ul>
-                <button type="button">See project</button>
+
             </article>
         </section>
 
```

Comment:
- [ ] Kindly adjust the background image of your about section to look like how we have it on the Figma design.
- [ ] You also need to work on the margin left of this section to match what we have on the Figma design

### Your Design
![your-about-section](https://user-images.githubusercontent.com/26597432/236642336-c1d5ae0b-da2c-4570-9fb6-f947e1561d83.png)

### Figma Design
![temp-about-section](https://user-images.githubusercontent.com/26597432/236642350-fac2d8d7-ccc6-4764-9c65-efe5725d7937.png)